src/dotnet/commands/dotnet-new/CSharp_Console/Program.cs

```diff
@@ -1,12 +1,9 @@
 ﻿using System;
 
-namespace ConsoleApplication
+class Program
```

Comment:
This change should be ported into https://github.com/dotnet/cli/tree/feature/msbuild/src/dotnet/commands/dotnet-new/CSharp_MSBuild as well.